pkg/plugin/driver/job/job_test.go

```diff
@@ -1,9 +1,12 @@
 package job
 
 import (
+	"crypto/sha1"
```

Comment:
I don't see where these fingerprints are being originally calculated, but we should use SHA-256 for these if possible.  SHA1 is generally deprecated for this use case: https://blog.qualys.com/ssllabs/2014/09/09/sha1-deprecation-what-you-need-to-know

Reply:
@mattmoyer it's literally only used in the tests, so it being faster is an advantage